monitoring/publishers/mysql_publisher_adapter.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from publisher import IPublisher
+from publishers.database import Build
+
+
+class MySQLPublisherAdapter(IPublisher):
+
+    def __init__(self, host, username, password, database):
+        connection_string = self.get_connection_string(host, username, password, database)
+        self.session = self.get_database_session(connection_string)
+
+
+    def publish(self, build_stats):
+        build = Build.from_build_stats(build_stats)
+        self.session.add(build)
+        self.session.commit()
+
+
+    @classmethod
+    def get_connection_string(cls, host, username, password, database):
```

Comment:
get_connection_string - same comment as for get_build_stats_endpoint. I think we should make it 1. private and 2. tested via Publisher API only.